src/Microsoft.Health.Fhir.Core/Features/Search/SearchValues/StringSearchValue.cs

```diff
@@ -53,6 +60,23 @@ public void AcceptVisitor(ISearchValueVisitor visitor)
             visitor.Visit(this);
         }
 
+        /// <inheritdoc />
+        public int Compare(ISearchValue searchValue)
+        {
+            if (searchValue == null)
+            {
+                throw new ArgumentException("Value to be compared to cannot be null");
+            }
+
+            var otherValue = searchValue as StringSearchValue;
+            if (otherValue == null)
+            {
+                throw new ArgumentException($"Value to be compared should be of type {typeof(StringSearchValue)}");
+            }
+
+            return string.Compare(ToString(), otherValue.ToString(), StringComparison.CurrentCultureIgnoreCase);
```

Comment:
A little unsure about whether `StringComparison.CurrentCultureIgnoreCase` is the current comparison mode here.

Reply:
Perhaps Invariant would be better. Though how does `Muller` compare to `Müller`? In FHIR they should be equal

Reply:
Good point. Updated the code to use `CompareOptions.IgnoreNonSpace` to ignore accents. Added a test case for the above strings as well.